src/ast/mod.rs

```diff
@@ -4050,6 +4051,13 @@ pub enum Statement {
         arguments: Vec<Expr>,
         options: Vec<RaisErrorOption>,
     },
+    /// Return (Mssql)
+    ///
+    /// for Functions:
+    /// RETURN scalar_expression
+    ///
+    /// See: https://learn.microsoft.com/en-us/sql/t-sql/statements/create-function-transact-sql
```

Comment:
Maybe we can move this to the `ReturnStatement` struct definition, and here we can only point to the statement like `Parse a [ReturnStatement]`? Thinking having it on the struct will be more visible as the struct can be reused/reached from more places

Reply:
Done 👍